### PR TITLE
Travis: use default node.js version

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
-    - 0.10
```
Comment (mightyiam) [inline .travis.yml]: This is a number. That seems likely the reason that, we end up testing against Node.JS `0.1`.

We could use a string, but I've good experience with leaving the default.

If/when it fails, we can either fix it or pin the version.

**Comment by mightyiam** [comment]
It works.

**Comment by mightyiam** [comment]
Please review.

**Comment by mightyiam** [comment]
It [only seems] as if it is using Node.js "0.1". Click on a build to see that it actually used `0.10`.

Still, I like this change.

**Comment by winhamwr** [comment]
:+1:

**Comment by mightyiam** [comment]
Thank you.

**Pull request merged**